pkg/commenting/service.go

```diff
@@ -0,0 +1,67 @@
+package commenting
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/TurnsCoffeeIntoScripts/jira-api-issue-resource/pkg/configuration"
+	"github.com/TurnsCoffeeIntoScripts/jira-api-issue-resource/pkg/http/rest"
+	"github.com/TurnsCoffeeIntoScripts/jira-api-issue-resource/pkg/service"
+	"net/http"
+)
+
+type ServiceAddComment struct {
+	issueId     string
+	commentBody string
+}
+
+func (s *ServiceAddComment) InitJiraAPI(params configuration.JiraAPIResourceParameters) (rest.JiraAPI, error) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported method ServiceAddComment.InitJiraAPI should have comment or be unexported](https://app.codacy.com/manual/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=5025522)